Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
Wouldn't it be more cleaner if you added the Linux change in here? Just saying...